common/changes/@stonecrop/atable/feat-sorting_and_filtering_2025-09-11-13-00.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@stonecrop/atable",
+      "comment": "Sorting and Filtering feature added by props",
+      "type": "none"
```

Let's change to "patch"


```suggestion
      "type": "patch"
```